lib/bench/either.bench.js

```diff
@@ -0,0 +1,34 @@
+var R = require('../..');
+
+var gt10 = x => x > 10;
+var even = x => x % 2 === 0;
+
+module.exports = {
+  name: 'either',
+  tests: {
+    'either(gt10, even, 101)': function() {
+      R.either(gt10, even, 101)
```

Comment:
[Either](https://github.com/ramda/ramda/blob/v0.22.1/src/either.js) has an arity of 2 and returns a function so calling it like this doesn't do what it looks like.